tests/e2e/specs/page-new-tab.test.ts

```diff
@@ -0,0 +1,14 @@
+import { canSwitchTheme } from '../helpers/theme';
+
+describe('Webextension New Tab', () => {
+  it('should open the extension page when a new tab is opened', async () => {
+    const extensionPath = await browser.getExtensionPath();
+    const newTabUrl = process.env.__FIREFOX__ === 'true' ? `${extensionPath}/new-tab/index.html` : 'chrome://newtab';
+
+    await browser.url(newTabUrl);
+
+    const appDiv = await $('.App');
```

Comment:
In all the docs of webdriverio, since v8 selector should be awaited.
I believe it's some kind of [bug of newly released v9](https://github.com/webdriverio/expect-webdriverio/issues/488).

Reply:
> In all the docs of webdriverio, since v8 selector should be awaited. I believe it's some kind of [bug of newly released v9](https://github.com/webdriverio/expect-webdriverio/issues/488).

Ok, but why you used it there, but in other tests, no?

Reply:
There were one case missing, otherwise it's used everywhere.
![image](https://github.com/user-attachments/assets/c5f61a09-0e98-40a2-8120-e79145cd3d8b)

Reply:
Ok, the v9 way of doing it's actually using `selector.getElement()`